nservicebus/outbox/index.md

```diff
@@ -79,8 +79,61 @@ In order to gradually convert an entire system from the DTC to the Outbox:
 WARNING: When verifying outbox functionality, it can sometimes be helpful to temporarily [stop the MSDTC Service](https://technet.microsoft.com/en-us/library/cc770732.aspx). This ensures that the Outbox is working as expected, and no other resources are enlisting in distributed transactions.
 
 
+## Message identity and idempotent processing
+
+If handlers are idempotent then outbox might not be required at all. When considering not using the outbox and implementing idempotent processing keep in mind that any generated message *must* apply the same message identifier if invoked more-then-once to have deduplication working on the infrastructure level or to solve this at the logical level by validating message data. [Read more on how to set a message identifier.](/nservicebus/messaging/message-identity.md)
+
+
+## Concurrency
+
+Outbox cannot guarantee that handlers or sagas are invoked exactly once. Outbox does not prevent concurrent attempts to process the same message but ensures that the outcome of the processing is persisted only once for resources that share the outbox transaction.
+
+The following scenarios might cause handlers to be invoked multiple times:
+
+- Sender sends multiple physical message with the same message identifier (submit order form)
```

Comment:
These bullet points read awkwardly. I *think* its because they are poorly split sentence fragments. Try re-writing them as full sentences on each bullet point. Use more than one sentence on each bullet if that makes it clearer. Once you have them reading clearly then you can explore removing wording and turning them into sentence fragments.